src/main/java/com/google/sps/data/Comment.java

```diff
@@ -1,7 +1,5 @@
 package com.google.sps.data;
-
-import java.util.Date;
-import java.util.Comparator;
+import java.util.*;
```

Comment:
Please use only the ones you need. Importing unneccessary libraries can make the binary larger which in turn makes it take more space and longer to build.

Reply:
Got it - just pushed out the PR fix!